src/tests/ftest/aggregation/dfuse_space_check.py

```diff
@@ -38,18 +39,24 @@ def get_nvme_free_space(self, display=True):
 
         return free_space_nvme
 
-    def check_aggregation(self):
-        """Check Aggregation for 240 secs max, else fail the test."""
-        counter = 1
-        while self.get_nvme_free_space() != self.space_before:
-            # try to wait for 4 x 60 secs for aggregation to be completed or
-            # else exit the test with a failure.
-            if counter > 4:
-                self.log.info("Free space when test terminated: %s",
-                              self.get_nvme_free_space())
-                self.fail("Aggregation did not complete as expected")
-            time.sleep(60)
-            counter += 1
+    def wait_for_aggregation(self, retries=4, interval=60):
+        """Wait for aggregation to finish.
+
+        Args:
+            retries (int, optional): number of times to retry.
+                Default is 4.
+            interval (int, optional): seconds to wait before retrying.
+                Default is 60.
+
+        """
+        for _ in range(retries):
+            current_space = self.get_nvme_free_space()
+            if current_space == self.initial_space:
+                return
+            time.sleep(interval)
+
+        self.log.info("Free space when test terminated: %s", current_space)
+        self.fail("Aggregation did not complete within {} seconds".format(retries * interval))
```

Comment:
(pylint-consider-using-f-string) Formatting a regular string which could be a f-string